### PR TITLE
Pipeline fixes

### PR DESCRIPTION
**Pull request by viktor-f**
**What this PR does / why we need it**: This adds a fix to the pipeline for a failing test. I also now runs kubespray at v2.17.1.

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

File: helmfile/values/falcosidekick.yaml.gotmpl [hunk 1]
```diff
@@ -1,24 +0,0 @@
-config:
```
Comment (OlleLarsson) [inline helmfile/values/falcosidekick.yaml.gotmpl:1]: Should this file be removed?

Comment (viktor-f) [reply helmfile/values/falcosidekick.yaml.gotmpl:1]: It is removed :slightly_smiling_face:

Comment (OlleLarsson) [reply helmfile/values/falcosidekick.yaml.gotmpl:1]: Because it's no longer needed or?

Comment (aarnq) [reply helmfile/values/falcosidekick.yaml.gotmpl:1]: It's now installed through the subchart included with falco, so we don't install it directly.
The old falcosidekick chart should also be removed from `charts/`.

Comment (OlleLarsson) [reply helmfile/values/falcosidekick.yaml.gotmpl:1]: Okay, then it makes sense.

Comment (viktor-f) [reply helmfile/values/falcosidekick.yaml.gotmpl:1]: Sorry, misunderstood your question. 

@aarnq I was thinking that the folder can be removed at the same time as https://github.com/elastisys/compliantkubernetes-apps/blob/b3c601e94de5e95ded0da6d2a8c2e0524f829c3c/helmfile/50-applications.yaml#L470 after the next release. But let me know if you would rather have me remove the folder now.

Comment (aarnq) [reply helmfile/values/falcosidekick.yaml.gotmpl:1]: @viktor-f The folder might as well be deleted now along with the values file. As long as `installed: false` is set then it won't check if either exists.

**Review by crssnd — APPROVED** [review]

**Review by OlleLarsson — APPROVED** [review]

**Review by Xartos — APPROVED** [review]
LGTM! Just remember to revert the test-branch names and stuff

**Pull request merged**